src/ceph-disk/ceph_disk/main.py

```diff
@@ -665,6 +665,19 @@ def get_dev_size(dev, size='megabytes'):
         os.close(fd)
 
 
+def is_diskdevice
```

Comment:
Right. We either have files, directories or devices in the sense that they are in /dev or are special files.